docs/6.x/cluster.md

```diff
@@ -208,6 +208,41 @@ $ curl -sk https://localhost:7575 | python -m json.tool
 
 In this case the response HTTP status code will be `503 Service Unavailable`.
 
+### Cluster Status History
+
+Running `gravity status history` will display a history of changes to the 
+cluster status. 
+
```

Comment:
I would add in a command similar to Cluster Status

```
$ tsh --cluster=production ssh admin@master gravity status history
```

I would recommend adding a sentence as to When and Why to run this command. 

Lastly, I would add a quick FAQ on using this command to https://gravitational.com/gravity/docs/faq/#common-issues